src/liftN.js

```diff
@@ -31,6 +31,6 @@ module.exports = _curry2(function liftN(arity, fn) {
         throw _noArgsException();
     }
     return curryN(arity, function() {
-        return _foldl(_ap, _map(lifted, arguments[0]), _slice(arguments, 1));
+        return _foldl(_ap, map(lifted, arguments[0]), _slice(arguments, 1));
```

Comment:
Internal `_ap` is dynamically dispatched.  The only difference from the public one is that it's not curried.  At some point we should have a discussion about whether this style is worth it.  I don't see this proliferation of functions as being overly helpful.